bundles/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/CBusThreadPool.java

```diff
@@ -45,10 +45,10 @@ protected synchronized CGateThreadPoolExecutor CreateExecutor(@Nullable String n
     }
 
     public class CBusThreadPoolExecutor extends CGateThreadPoolExecutor {
-        private final ThreadPoolExecutor threadPool;
+        private final ExecutorService threadPool;
 
         public CBusThreadPoolExecutor(@Nullable String poolName) {
-            threadPool = (ThreadPoolExecutor) ThreadPoolManager.getPool("binding.cbus-" + poolName);
+            threadPool = (ExecutorService) ThreadPoolManager.getPool("binding.cbus-" + poolName);
```

Comment:
I don't think casting is needed as `getPool` returns `ExecutorService`:
https://www.openhab.org/javadoc/latest/org/openhab/core/common/threadpoolmanager#getPool(java.lang.String)
?